build failed

@claude Run npx --no-install next build
  ▲ Next.js 14.2.13

   Creating an optimized production build ...
Generated 12 documents in .contentlayer
 ✓ Compiled successfully
   Linting and checking validity of types ...
Failed to compile.

./src/components/analyses/gemeentelijke-investeringen/InvesteringenDistributionPlot.tsx:134:74
Type error: Property 'percentileLabel' does not exist on type '{ binIndex: number; min: number; max: number; count: number; isHighlighted: boolean; }'.

  132 |             return (
  133 |                 <div className="bg-background border border-border rounded-md shadow-lg p-3 text-sm">
> 134 |                     <p className="font-bold mb-1">Percentiel Groep ({bin.percentileLabel})</p>
      |                                                                          ^
  135 |                     <p className="text-muted-foreground">
  136 |                         Bereik: {valueFormatter(bin.min)} - {valueFormatter(bin.max)}
  137 |                     </p>
Error: Process completed with exit code 1.

--- a/embuild-analyses/src/components/analyses/gemeentelijke-investeringen/InvesteringenDistributionPlot.tsx
+++ b/embuild-analyses/src/components/analyses/gemeentelijke-investeringen/InvesteringenDistributionPlot.tsx
@@ -122,6 +122,8 @@ export function InvesteringenDistributionPlot({
                 max: number
                 count: number
                 isHighlighted: boolean
+                medianValue: number
+                percentileLabel: string
             }
         }>
     }

